packages/api-sdk/.speakeasy/gen.lock

```diff
@@ -5,8 +5,8 @@ management:
   docVersion: 1.0.0
   speakeasyVersion: 1.551.0
   generationVersion: 2.610.0
-  releaseVersion: 0.0.7
-  configChecksum: 78aada5706e22d1c5bd9f7be82eea621
+  releaseVersion: 0.0.8
+  configChecksum: c609247661631dec1ea46cbcbb9adaff
```

Comment:
do we need to commit all of these `packages/api-sdk` changes?

Reply:
Makes sense! I overlooked it. Removing....